patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -207,6 +213,18 @@
 +    protected ThreadLocal<EntityPlayer> harvesters = new ThreadLocal();
 +    private ThreadLocal<IBlockState> silk_check_state = new ThreadLocal();
 +    protected static java.util.Random RANDOM = new java.util.Random(); // Useful for random things without a seed.
++
++    /**
++     * Checks if the block ticks randomly
++     *
++     * @param state Block state
++     * @return True if the block ticks randomly
++     */
++    public boolean getTickRandomly(IBlockState state)
```

Comment:
I thought about that, but `ExtendedBlockStorage` doesn't have a world instance (AFAIK).